KubeArmor/core/containerdHandler.go

```diff
@@ -162,6 +163,10 @@ func (ch *ContainerdHandler) GetContainerInfo(ctx context.Context, containerID s
 		container.NamespaceName = "container_namespace"
 	}
 
+	container.Owner.Name = OwnerInfo[container.EndPointName]["owner.controllerName"]
```

Comment:
@daemon1024 if we do not get the `OwnerInfo` initially, do we send a few logs without the owner, or do we wait until we get the info? If we wait, should we add sleep?

I don't find any way by which we can trigger the code to get the `OwnerInfo`. It will occur automatically, but what do we do until we do not get the info?